### ISSUE TITLE
syslinux-utils required for 16.04 (host system)

### ISSUE THREAD

**Issue by mbr0wn** [gnuradio/gnuradio-livesdr#14]

**Comment by mbr0wn** [comment]
After running make deps:

```
[ERROR] /usr/bin/isohybrid not found or not executable!
[INFO] Try running make deps.
```

**Comment by jmcorgan** [comment]
I'll look into this, thanks.

**Comment by jmcorgan** [comment]
Indeed, it just looks like we need to add syslinux-utils to the deps packages.

**Comment by jmcorgan** [comment]
This was fixed in e95210a
